File: doc/Examples_SPL/Expresiones/Expresiones/Eval/multEval.cs
using System;
using Expresiones;


namespace Expresiones
{
    
    public class MultEval : BinaryOperatorEval, Mult
    {
        protected static IExpressionEval exp_izquierda;
        protected static IExpressionEval exp_derecha;
        /**
         * Constructor
         * */
        public MultEval(IExpressionEval izq, IExpressionEval derch)
        {
            exp_izquierda = izq;
            exp_derecha = derch;
        }
        /**
        * Método que retorna el resultado de evaluar la expresion
        * */
        public int eval()
        {
            return exp_izquierda.eval() * exp_derecha.eval();
        }
    }
}

File: doc/Examples_SPL/expressionsCaesarJ/src/eval/MultEval.java
cclass eval.Eval;

public cclass MultEval extends BinaryOperatorEval & Mult {
	protected ExpressionEval op1;
	protected ExpressionEval op2;
	public MultEval(ExpressionEval op1, ExpressionEval op2){
		this.op1=op1;
		this.op2=op2;
	}
	public int eval(){
		return this.op1.eval()*this.op2.eval();
	}
}
